packages/react-notion-x/src/components/asset-wrapper.tsx

```diff
@@ -33,7 +33,6 @@ export const AssetWrapper: React.FC<{
       className={cs(
         'notion-asset-wrapper',
         `notion-asset-wrapper-${block.type}`,
-        value.format?.block_full_width && 'notion-asset-wrapper-full',
```

Comment:
i'm pretty sure this is used for certain full width blocks and won't render faithfully without this